Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+    :return: A CommandResults object containing the list of object types as output, and a human-readable markdown table.
+    """
+    schema_id = args.get('schema_id')
+    query = args.get('query')
+    exclude = args.get('exclude')
+    limit = args.get('limit', 50)
+    all_results = argToBoolean(args.get('all_results', False))
+
+    # build outputs
+    res = client.get_object_type_list(schema_id, query, exclude)
+    object_types = convert_keys_to_pascal(list(res), {'id': 'ID', 'parentObjectTypeId': 'ParentTypeID'})
+    if not all_results:
+        limit = int(limit)
+        object_types = object_types[:limit]
+    outputs = [{k: v for k, v in ot.items() if k != 'Icon'} for ot in object_types]
+
+    # build readable outputs
+    hr_headers = ['ID', 'Name', 'ParentTypeID', 'AbstractObjectType']
+
+    return CommandResults(
+        outputs_prefix=f'{INTEGRATION_OUTPUTS_BASE_PATH}.ObjectType',
+        outputs_key_field='ID',
+        outputs=outputs,
+        readable_output=tableToMarkdown('Object Types', outputs, headers=hr_headers, removeNull=True)
+    )
+
+
+def jira_asset_object_type_attribute_list_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+        Retrieves a list of attributes for a specific Jira asset object type with various filtering and sorting options.
+
+        :param client: Client object for performing API requests.
+        :param args: A dictionary of command arguments:
+            - 'object_type_id': The ID of the object type to retrieve attributes for.
+            - 'limit': The maximum number of attributes to return. Defaults to 50.
+            - 'all_results': A boolean indicating whether to return all results or to respect the limit. Defaults to False.
+            - Additional boolean arguments for filtering and sorting:
+                * 'order_by_name'
+                * 'include_value_exist'
+                * 'exclude_parent_attributes'
+                * 'include_children'
+                * 'order_by_required'
+        :return: A CommandResults object containing the list of attributes as output, and a human-readable markdown table.
+    """
+    object_type_id = args.get('object_type_id')
+    limit = args.get('limit', 50)
+    all_results = args.get('all_results', False)
+
+    # build outputs
+    res = client.get_object_type_attributes(
+        object_type_id=object_type_id,
+        order_by_name=argToBoolean(args.get('order_by_name', False)),
+        query=args.get('query'),
+        include_value_exist=argToBoolean(args.get('include_value_exist', False)),
+        exclude_parent_attributes=argToBoolean(args.get('exclude_parent_attributes', False)),
+        include_children=argToBoolean(args.get('include_children', False)),
+        order_by_required=argToBoolean(args.get('order_by_required', False))
+    )
+
+    outputs = clean_object_attributes(list(res))
+
+    if not all_results:
```

Comment:
the limit cant be passed to the api on this 1?

Reply:
no